app/assets/javascripts/application.js

```diff
@@ -85,6 +85,8 @@ $(function() {
   $('a[href*=#]:not([href=#])').click(function() {
     if (location.pathname.replace(/^\//,'') == this.pathname.replace(/^\//,'') && location.hostname == this.hostname) {
       var target = $(this.hash);
+      e.preventDefault()
```

Comment:
I found this on Stack Overflow and it worked. But I don't know what Hound wants me to do here.